src/core/file_sys/patch_manager.cpp

```diff
@@ -80,16 +80,6 @@ VirtualDir PatchManager::PatchExeFS(VirtualDir exefs) const {
     if (exefs == nullptr)
         return exefs;
 
-    if (Settings::values.dump_exefs) {
-        LOG_INFO(Loader, "Dumping ExeFS for title_id={:016X}", title_id);
-        const auto dump_dir =
-            Core::System::GetInstance().GetFileSystemController().GetModificationDumpRoot(title_id);
-        if (dump_dir != nullptr) {
-            const auto exefs_dir = GetOrCreateDirectoryRelative(dump_dir, "/exefs");
-            VfsRawCopyD(exefs, exefs_dir);
-        }
-    }
-
     const auto& installed = Core::System::GetInstance().GetContentProvider();
```

Comment:
It's a little out of scope but we might as well clean up the function while we're at it
```suggestion
    auto &system = Core::System::GetInstance();
    auto &fs_controller = system.GetFileSystemController();
    const auto& installed = system.GetContentProvider();
```
Just so we can eliminate calling `Core::System::GetInstance()` everywhere here

Reply:
Good idea, I've added a new commit that removes the redundant `Core::System::GetInstance()` from the function.